docs/tags.rst

```diff
@@ -114,7 +114,8 @@ The following values are take from the ``BoundField``:
 - id_for_label
 - label
 - name
-- value
+- value (value of the field converted to string)
+- raw_value (e.g. True/False for BooleanField or datetime.datetime for DateTimeField)
```

Comment:
Is this always a True/False? What happens when a form is submitted with validation errors: what is `raw_value` then?